interfaces/builtin/all_test.go

```diff
@@ -200,7 +200,12 @@ var allGoodDefiners = []reflect.Type{
 
 // Check that each interface defines at least one definer method we recognize.
 func (s *AllSuite) TestEachInterfaceImplementsSomeBackendMethods(c *C) {
+	exceptions := map[string]bool{"registry": true}
 	for _, iface := range builtin.Interfaces() {
+		if exceptions[iface.Name()] {
+			continue
+		}
+
```

Comment:
What makes the registry interface different enough from other interfaces to require an exception?

Reply:
It doesn't add any snippets for apparmor, seccomp, etc so without this exception the test would fail. The interface is only used to ensure that snaps have access to a registry view

Reply:
What about something like the snap_refresh_observe interface which, afaict, doesn't do anything at all except that it's used as a marker interface for some notice types and API calls?

Not a blocker here, I'm just trying to understand a bit more the spread of how different interfaces are handled :)

Reply:
That embeds `commonInterface` which defines some of these backends (as no-ops by default). I've changed this branch to do the same, there should be no behavioural change. Thanks!